### ISSUE TITLE
Docker rock for survival package

### ISSUE THREAD

**Issue by tombisho** [datashield/docker-rock#2]
Is this the repo I should use to create a Rock image for survival analysis?

**Comment by ymarcon** [comment]
See https://github.com/obiba/docker-rock-demo/tree/survival

**Comment by tombisho** [comment]
Thanks, I will certainly make use of that 😀 

Maybe my question should have been, is this the correct home for a survival analysis image, rather than one that is in a "demo" repo?

**Comment by ymarcon** [comment]
dsSurvival package is not in the official [Datashield's Github](https://github.com/datashield). You should build and publish your own image, using this example. If you do so, I'll use it in opal-demo.obiba.org in place of this demo one.

**Comment by StuartWheater** [comment]
I was wondering what is the plan if an docker image is needed which contains both dsMediation, dsExposome and dsSurvival?
Do we need a Community Docker Hub account?

**Comment by ymarcon** [comment]
We can't do all combinations, then it is up to the consortium to make their own images. And they would probably use a private docker repo, just like Inesctec does with Coral for Recap.

**Comment by tombisho** [comment]
Yes although it might be nice for consortia to share to reduce work?

**Comment by tombisho** [comment]
And yes I can see that I should create my image in the Github where the package has been created...

**Comment by ymarcon** [comment]
You could have the `Dockerfile` within your R package source repo, for user reference. Just declare it in a `.Rbuildignore` file so that it does not get packaged.

**Comment by tombisho** [comment]
Yes that sounds like a good idea.

And then if I build an image, can it live in the DataSHIELD docker hub
account (like for omics and exposome)? Or is this where we need some kind
of audit?

**Comment by ymarcon** [comment]
omics and exposome should be in https://hub.docker.com/u/brgelab space. You should have your own as well.

**Comment by tombisho** [comment]
Ok great - I'll do that

**Comment by StuartWheater** [comment]
dsSurvival is in new profile "dolomite-beach"
